_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,175 @@
+    - For each of the unique dimension values combinations , we also pre-compute the aggregated values for all the metrics such as `avg(size)` and `count(requests)` 
+
+Each node in the Star Tree points to a range of star-tree documents.
+A node is further split into child nodes based on [max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/star-tree/#star-tree-configuration-parameters).
+The number of documents a leaf node points to is than or equal to `max_leaf_docs`. This ensures the maximum number of documents that gets traversed to get to the aggregated value is at most `max_leaf_docs`, thus providing predictable latencies.
+
+There are special nodes called `star nodes (*)` which helps in skipping non-competitive nodes and also in fetching aggregated document wherever applicable during query time.
+
+The figure contains three examples explaining the Star Tree traversal during query: 
+- Compute average request size aggregation with Terms query where port equals 8443 and status equals 200 (Support for Terms query will be added in upcoming release)
+- Compute count of requests aggregation with Term query where status equals 200 (query traverses through * node of `port` dimension since `port` is not present as part of query) 
+- Compute average request size aggregation with Term query where port equals 5600 (query traverses through * node of `status` dimension since `status` is not present as part of query). 
+<br/>The second and third examples uses star nodes.
+
+
+## When to use star tree index
+You can be use Star Tree index to perform faster aggregations with a constant upper bound on query latency.
+- Star Tree natively supports multi field aggregations
+- Star Tree index will be created in real time as part of regular indexing, so the data in Star Tree will always be up to date with the live data.
+- Star Tree index consolidates the data and hence is a storage efficient index which results in efficient paging and fraction of IO utilization for search queries. 
+
+## Considerations
+- Star Tree index ideally should be used with append-only indexes, as updates or deletes are not accounted in Star Tree index.
+- Star Tree index will be used for aggregation queries only if the query input is a subset of the Star Tree configuration of dimensions and metrics
+- Once star-tree index is enabled for an index, you currently cannot disable it. You have to reindex without the star-tree mapping to remove star-tree from the index.
+    - Changing Star Tree configuration will also require a re-index operation.
+- [Multi-values/array values]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/index/#arrays) are not supported
+- Only [limited queries and aggregations](#supported-query-and-aggregations) are supported with support for more coming in future
+- The cardinality of the dimensions should not be very high (like `_id` fields), otherwise it leads to storage explosion and higher query latencies.
```

Comment:
```suggestion
- The cardinality of the dimensions should not be very high (like `_id` fields), otherwise it leads to high storage usage and higher query latency.
```